### PR TITLE
Update logger.py to create only one root logger that emits messages to de-dup log messages.

### PR DESCRIPTION
**Pull request by maniarathi**
The main change here is the modifications to logging.py which configures a root logger once with the default handler and then in each file that requires logging, creates a logger than will be a child of the root logger. However since the root logger is the only logger with a handler, the payload will only be emitted once (Python default is to propagate the payload up the hierarchy and emit when passing through any logger with a handler attached).

Also contains some reordering of imports and general cleanup because now it looks pretty and I like things alphabetical. Happy to revert though if you think it clogs up the PR.

### CODE REVIEW

**Comment by parthshahva** [comment]
@maniarathi Could you please make the PR name more descriptive? This will be used in the squash and merge

File: upload/docker_images/validator/validator_harness.py [hunk 1]
```diff
@@ -1,25 +1,22 @@
...
+from urllib3.util import parse_url
 
-from upload.common.validation_event import ValidationEvent
-from upload.common.logging import get_logger
 from upload.common.upload_api_client import update_event
+from upload.common.validation_event import ValidationEvent
 
-
-logger = get_logger(f"CHECKSUMMER [{os.environ.get('AWS_BATCH_JOB_ID')}]")
+logger = logging.getLogger(f"CHECKSUMMER [{os.environ.get('AWS_BATCH_JOB_ID')}]")
```
Comment (parthshahva) [inline upload/docker_images/validator/validator_harness.py]: This is invoked in the docker image that runs in batch. What is the consequence of not calling configure_logging first?

File: upload/docker_images/checksummer/checksummer.py [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
-from upload.common.logging import get_logger
-from upload.common.dss_checksums import DssChecksums
 from upload.common.checksum_event import ChecksumEvent
+from upload.common.dss_checksums import DssChecksums
 from upload.common.upload_api_client import update_event
 from upload.common.upload_config import UploadConfig
 
-logger = get_logger(f"CHECKSUMMER [{os.environ.get('AWS_BATCH_JOB_ID')}]")
+logger = logging.getLogger(f"CHECKSUMMER [{os.environ.get('AWS_BATCH_JOB_ID')}]")
```
Comment (parthshahva) [inline upload/docker_images/checksummer/checksummer.py]: same question about not invoking configure_logging in the docker image

**Comment by maniarathi** [comment]
> @maniarathi Could you please make the PR name more descriptive? This will be used in the squash and merge

Done

File: upload/docker_images/validator/validator_harness.py (hunk 1 above)
Comment (maniarathi) [reply upload/docker_images/validator/validator_harness.py]: Talked with Andrey a bit and seems pretty reasonable and makes it super simple moving forward so did it. Thanks for the suggestion!

File: upload/docker_images/checksummer/checksummer.py (hunk 2 above)
Comment (maniarathi) [reply upload/docker_images/checksummer/checksummer.py]: Ditto from above.

**Pull request merged**